HWAlevel/Moudule3/Module3_HW5/Read.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+
+namespace Module3_HW5
+{
+    public sealed class Read
+    {
+        public string hello, world = "";
```

Comment:
please use properties. Your syntax is not used and is so rare.
public string Hello { get; set; }
public string World { get; set; }